Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM registry.redhat.io/rhel8/go-toolset:1.15 AS builder
+FROM registry.redhat.io/rhel8/go-toolset:1.16.7 AS builder
```

Comment:
In Content Service `1.16` version was used. I think it would be better to use 1.16 instead of an specific one, but I'm not an expert on Golang versioning